docs/conf.py

```diff
@@ -29,6 +32,36 @@ def make_pypi_svg(version: str) -> None:
         f.write(svg)
 
 
+def replace_pr_numbers_with_links(content: str) -> str:
+    """Replaces all PR numbers with the corresponding GitHub link."""
+
+    base_url = "https://github.com/psf/black/pull/"
+    pr_num_regex = re.compile(r"\(#(\d+)\)")
+
+    def num_to_link(match: re.Match) -> str:
+        full_match, number = match.group(0, 1)
+        url = f"{base_url}{number}"
+        return f"[{full_match}]({url})"
+
+    return pr_num_regex.sub(num_to_link, content)
+
+
+def handle_include_read(
+        app: Sphinx,
```

Comment:
I guess we're not formatting this file with Black? Let's fix that in a separate PR though.